### ISSUE TITLE
Perhaps does not require transpilation

### ISSUE THREAD

**Issue by mightyiam** [Urucas/sha-bang#5]
Depends on what engines you're targeting, this may not require a transpilation step.

**Comment by vrunoa** [comment]
Yeah I can avoid transpilation. I'll update and fix all this issues in a new version of the module
